src/OpenLoco/src/Windows/MapWindow.cpp

```diff
@@ -380,8 +1041,8 @@ namespace OpenLoco::Ui::Windows::MapWindow
     static void getScrollSize(Window& self, [[maybe_unused]] uint32_t scrollIndex, uint16_t* scrollWidth, uint16_t* scrollHeight)
     {
         self.callPrepareDraw();
-        *scrollWidth = kMapColumns * 2;
-        *scrollHeight = kMapRows * 2;
+        *scrollWidth = kRenderedMapWidth;
```

Comment:
cool to see your behaviour is identical to vanilla, and this would technically be an improvement, but it always annoyed me the map never could scroll to show the left-most and top-most tiles, they've always been cut off. glad we can change/fix that now

![image](https://github.com/OpenLoco/OpenLoco/assets/7483209/df2375b3-841f-4b72-b15b-00ed6ac77635)

Reply:
Definitely worth pursuing in a follow-up PR, I agree.